src/Components/FooterNavigation.js

```diff
@@ -0,0 +1,70 @@
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+import formicon from '../assets/form-icon.svg';
+import listicon from '../assets/list-icon.svg';
+import mapicon from '../assets/map-icon.svg';
+
+export default function FooterNavigation({ isStatic }) {
+  return (
+    <Nav isStatic={isStatic}>
+      <NavBox>
+        <NavLinkStyled exact to="/" activeClassName="active" className="link">
+          <img src={formicon} alt="formicon" />
+        </NavLinkStyled>
+        <NavLinkStyled
+          to="/location-list"
+          activeClassName="active"
+          className="link"
+        >
+          <img src={listicon} alt="listicon" />
+        </NavLinkStyled>
+        <NavLinkStyled to="/map" activeClassName="active" className="link">
+          <img src={mapicon} alt="mapicon" />
+        </NavLinkStyled>
+      </NavBox>
+    </Nav>
+  );
+}
+
+const Nav = styled.nav`
+  position: ${(props) => (props.isStatic ? 'static' : 'fixed')};
+  bottom: 0;
+  padding: 1.25rem;
+  width: 100%;
+  z-index: 100;
+`;
+
+const NavBox = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  background-color: var(--primary);
+  border-radius: 0.375rem;
+  color: transparent;
+  height: 2.75rem;
+
+  .active {
+    opacity: 0.9;
+  }
+`;
+
+const NavLinkStyled = styled(NavLink)`
+  display: inline-flex;
+  //border: 0.031rem solid var(--border-dark);
+  box-shadow: 0 2px 0.75rem hsla(213, 52%, 20%, 0.2);
+  border-radius: 3rem;
+  padding: 12px 14px 10px 14px;
+  text-decoration: none;
+  background: transparent;
+  color: transparent;
+  opacity: 0.5;
+
+  :hover {
+    background: var(--secondary);
+  }
+
+  img:hover {
+    background: var(--secondary);
+    border-radius: 3rem;
```

Comment:
When I hover over your icons the icons seem to be cut, I think thats because of your big border-radius. If this is a style choice please ignore my comment :)